test/AuthBridgeExecutor.t.sol

```diff
@@ -0,0 +1,543 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+
+import { AuthBridgeExecutor } from 'src/executors/AuthBridgeExecutor.sol';
+import { IExecutorBase }      from 'src/interfaces/IExecutorBase.sol';
+
+contract DefaultPayload {
+    event TestEvent();
+    function execute() external {
+        emit TestEvent();
+    }
+}
+
+contract PayablePayload {
+    event TestEvent();
+    function execute() external payable {
+        emit TestEvent();
+    }
+}
+
+contract RevertingPayload {
+    function execute() external pure {
+        revert("An error occurred");
+    }
+}
+
+contract AuthBridgeExecutorTest is Test {
```

Comment:
Lets split these into base and contracts by function to make the tests easier to navigate

Reply:
done